src/migrations/1729857146775-item-category-to-item.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Migrations1729857146775 implements MigrationInterface {
+  name = 'language-category-to-item-1729857146775';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    // we ignore english category because default item lang is already english
+    for (const [lang, key] of [
+      ['french', 'fr'],
+      ['german', 'de'],
+      ['arabic', 'ar'],
+      ['italian', 'it'],
+    ]) {
+      // change item lang based on category lang
+      await queryRunner.query(
+        `UPDATE item SET lang='${key}' 
+        WHERE path IN (
+          SELECT item_path FROM item_category 
+          LEFT JOIN category ON category_id = category.id
+          WHERE category.name = '${lang}'
+        )`,
+      );
+
+      // remove language category from db
+      // cascade delete relatede item_category
+      await queryRunner.query(`DELETE FROM category where type='language'`);
+    }
+  }
+
+  public async down(): Promise<void> {}
```

Comment:
I guess it makes sens that we can't really have a down migration for this step since we are potentially loosing data. I am wondering however if we should still try to have a migration that at least adds back the languages in the categories ?